core/src/impl/Cabana_CartesianGrid.hpp

```diff
@@ -75,11 +85,25 @@ class CartesianGrid
         num_z = _nz;
     }
 
+    // Get the number of cells in a given direction.
+    KOKKOS_INLINE_FUNCTION
```

Comment:
Did we want to wrap this `Kokkos_xxxx` with `Cabana_xxxx`?

Reply:
I'm torn if we should propagate the Cabana macros to implementation details